trino-rest-github/src/main/java/pl/net/was/rest/github/Sync.java

```diff
@@ -84,7 +86,8 @@ public static void main(String[] args)
                 "LOG_LEVEL", "INFO",
                 "EMPTY_INSERT_LIMIT", "1",
                 "CHECK_STEPS_DUPLICATES", "false",
-                "CHECK_ARTIFACTS_DUPLICATES", "false");
+                "CHECK_ARTIFACTS_DUPLICATES", "false",
+                "QUERY_RETRY_DELAY_MS", "500");
```

Comment:
Figured millisecond granularity would be good, but lemme know if just using seconds is preferred